safe/definitions/hazard_classifications.py

```diff
@@ -123,9 +135,9 @@
         exposure_structure
     ]
 }
-
-earthquake_mmi_hazard_classes = {
-    'key': 'earthquake_mmi_hazard_classes',
+# do not use these old classes until we can trace the origin
+earthquake_mmi_old_hazard_classes = {
+    'key': 'earthquake_mmi_old_hazard_classes',
```

Comment:
Do we really need to change the `key` here? I saw @Charlotte-Morgan add new classes but with different `key`. So, it should be fine. And I think it's better to change the `name` since the user will not know about the `key` or the variable name.
Just bear in mind, changing the `key` means we need to update all layer's metadata that we already assigned with that key.